src/status_im/wallet/choose_recipient/core.cljs

```diff
@@ -94,19 +96,23 @@
   {:events [:wallet.send/set-recipient ::recipient-address-resolved]}
   [{:keys [db]} recipient]
   (let [chain (ethereum/chain-keyword db)]
-    (if (ens/is-valid-eth-name? recipient)
+    (cond
+      (ethereum/address? recipient)
+      (let [checksum (eip55/address->checksum recipient)]
+        (if (eip55/valid-address-checksum? checksum)
+          {:db       (-> db
```

Comment:
We could use `(update db :wallet/prepare-transaction assoc :to checksum :modal-opened? false)` here